src/ring_theory/ideal/local_ring.lean

```diff
@@ -32,8 +32,7 @@ universes u v w u'
 
 variables {R : Type u} {S : Type v} {T : Type w} {K : Type u'}
 
-/-- A commutative ring is local if it has a unique maximal ideal. Note that
-  `local_ring` is a predicate. -/
+/-- A semiring is local if it has a unique maximal ideal. Note that `local_ring` is a predicate. -/
```

Comment:
I mean that now the definition is not that there is only one maximal ideal.

Reply:
Thanks I see what you mean.

Having unique maximal ideal ii not the definition itself even in the previous file. I guess that the previous author wrote such docs as the file show the equivalence of the definitions soon after. Should we write the actual definition here?

Reply:
Yes, you can add a comment saying it is equivalent to the usual one.

Reply:
OK!